packages/@nucleus/tests/dummy/app/templates/components/playground/props.hbs

```diff
@@ -0,0 +1,45 @@
+{{#each properties as |property|}}
+  <div class="item">
+    <label for="{{property.name}}" class="item--label">
+      {{property.name}}
+    </label>
+
+    {{!-- select box --}}
+    {{#if property.select}}
+      <select
+        onchange={{action 'handleChange' property}}
+        name="{{property.name}}"
+        id="{{property.name}}"
+        value="{{property.value}}">
+        {{#each property.types as |type|}}
+          <option value="{{type}}">{{type}}</option>
+        {{/each}}
+      </select>
+    {{/if}}
+
+    {{!-- input --}}
+    {{#if property.input}}
+      <Input
```

Comment:
Also, can we have some basic styling for input to make it appear editable? I know we'll replace this with nucleus-input in the future, but it doesn't look like a text field at all now.